docs/rhel4edge_iso.md

```diff
@@ -41,9 +41,10 @@ Note that the command deletes various user and system data, including:
 Run the build script without arguments to see its usage.
 ```bash
 ./scripts/image-builder/build.sh
-Usage: build.sh <-pull_secret_file path_to_file> [-ostree_server_name name_or_ip] [-custom_rpms /path/to/file1.rpm,...,/path/to/fileN.rpm]
+Usage: build.sh <-pull_secret_file path_to_file> [-ostree_server_name name_or_ip] [-lvm_sysroot_size num_in_MB] [-custom_rpms /path/to/file1.rpm,...,/path/to/fileN.rpm]
```

Comment:
Nit: Do the angled brackets signify a required argument?

Reply:
Yes, <> brackets are meant to denote required args. Only pull secret is required for this script.